freeipa/src/main/resources/freeipa-salt/salt/upgradeccm/init.sls

```diff
@@ -0,0 +1,28 @@
+create_upgrade_ccm_script:
+  file.managed:
+    - name: /opt/salt/scripts/upgrade-ccm.sh
+    - user: root
+    - group: root
+    - mode: 700
+    - makedirs: True
+    - source: salt://{{ slspath }}/scripts/upgrade-ccm.sh
+
+call_upgrade_ccm:
+  cmd.run:
+    - name: /opt/salt/scripts/upgrade-ccm.sh | tee -a /var/log/upgrade-ccm.log
```

Comment:
could you add this file to the logcollection?

Reply:
adding